python/ray/serve/schema.py

```diff
@@ -978,6 +978,92 @@ class ReplicaDetails(ServeActorDetails, frozen=True):
     )
 
 
+@PublicAPI(stability="alpha")
+class MetricsHealth(str, Enum):
+    HEALTHY = "healthy"
+    DELAYED = "delayed"
+    UNAVAILABLE = "unavailable"
+
+
+@PublicAPI(stability="alpha")
+class ScalingSource(str, Enum):
+    DEFAULT = "default"
+    CUSTOM = "custom"
+    EXTERNAL = "external"
+
+
+@PublicAPI(stability="alpha")
+class ScalingStatus(str, Enum):
+    SCALING_UP = "scaling_up"
+    SCALING_DOWN = "scaling_down"
+    STABLE = "stable"
+
+
+@PublicAPI(stability="alpha")
+class ScalingDecision(BaseModel):
+    """One autoscaling decision with minimal provenance."""
+
+    timestamp_s: float = Field(
+        ..., description="Unix time (seconds) when the decision was made."
+    )
+    source: ScalingSource = Field(
+        ..., description="Who made this decision: default/custom/external."
+    )
+    reason: str = Field(
+        ..., description="Short, human-readable reason for the decision."
+    )
+    from_replicas: int = Field(
+        ..., ge=0, description="Replica count before the decision."
+    )
+    to_replicas: int = Field(..., ge=0, description="Replica count after the decision.")
+    policy: Optional[str] = Field(
+        None, description="Policy name or identifier (if applicable)."
+    )
+    metrics: Optional[Dict[str, Any]] = Field(
+        None, description="Snapshot of metrics consulted when making the decision."
+    )
+
+
+@PublicAPI(stability="alpha")
+class DeploymentAutoscalerView(BaseModel):
```

Comment:
`DeploymentAutoscalerView` -> `DeploymentAutoscalingDetail`

@zcin do you have better suggestion here?